deezer.py

```diff
@@ -16,48 +16,91 @@ def env_constructor(loader, node):
 
 yaml.SafeLoader.add_constructor("!env", env_constructor)
 
-if __name__ == '__main__':
-    with open("config.yml", 'r') as f:
+
+async def build_track_entry(builder, el):
+    artist_label = f"Artist: {el['artist']['name']}"
+    album_label = f"Album: {el['album']['title']}"
+    explicit_label = "\nExplicit" if el["explicit_lyrics"] else ""
```

Comment:
we need to check, but I'm not sure if telegram allows 3 lines of description

Reply:
Telegram API does not specify max length or amount of lines in an article description, so we have to check that by running the actual code

Reply:
this is exacty what I meant :D

Reply:
just checked and it doesn't show up